script/build-wasm

```diff
@@ -78,8 +78,9 @@ fi
 if [ -z "$emcc" ] && [ -n "$docker" ]; then
   export PODMAN_USERNS=keep-id
   emcc="$docker run                      \
+  --platform linux/amd64                 \
```

Comment:
are you sure about this? or was this just a leftover remnant from testing on your own machine